### ISSUE TITLE
Please consider giving your service a more specific title

### ISSUE THREAD

**Issue by CottonEaster** [erpalma/lenovo-throttling-fix#88]
Thank you for your continuing work - work that compensates for Lenovo's shortcomings.

Would you consider renaming the service that your program creates? For, as you can imagine, I've more that one service installed to fix Lenovo incompetence. How, then, about creating a service called, not simply `lenovo_fix` but `lenovo_throttleFix`?

**Comment by erpalma** [comment]
I agree. I'm just concerned about breaking the existing installations!

**Comment by CottonEaster** [comment]
Is there some clever way around that? Perhaps you need a script that renamed existing stuff . .
